sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -1743,54 +1621,233 @@ or (calculatedDataPartnerFedralProvincialPDReceiptCost &gt; 0)</bpmn:conditionEx
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0x3es0p</bpmn:incoming>
+      <bpmn:outgoing>Flow_101k7op</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:sequenceFlow id="Flow_0x3es0p" sourceRef="Event_1vx49ib" targetRef="Event_1yp89qu" />
+    <bpmn:intermediateThrowEvent id="Event_0v38z6l" name="awardNetBCSLAmount1">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if provincialAwardNetBCSLAmount &#62; provincialAwardNetBGPDAmount&#10;then&#10;  provincialAwardNetBCSLAmount - provincialAwardNetBGPDAmount&#10;else&#10;provincialAwardNetBGPDAmount" target="provincialAwardNetBCSLAmount1" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0z7ptdx</bpmn:incoming>
+      <bpmn:outgoing>Flow_1p1lxb9</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:sequenceFlow id="Flow_08y7zg9" sourceRef="Event_1ox30qu" targetRef="Event_1y3t4n6" />
+    <bpmn:intermediateThrowEvent id="Event_1vx49ib" name="awardRemainingBCSL">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if &#10;  provincialAwardProvincialUnmetNeed &#62; provincialAwardExcessCSGVariation&#10;then&#10;  true&#10;else&#10;  false" target="provincialAwardprovincialunmetmorethanfederalgrant" />
+          <zeebe:output source="=provincialAwardAjustedProvincialAssessedNeed - provincialAwardNetBCSLPotentialAmount" target="provincialAwardProvincialUnmetNeed" />
+          <zeebe:output source="=if provincialAwardprovincialunmetmorethanfederalgrant = false&#10;then &#10;  provincialAwardProvincialUnmetNeed&#10;else &#10;  provincialAwardExcessCSGVariation" target="provincialAwardDisplacedProvincialUnmetNeed" />
+          <zeebe:output source="=if&#10;provincialAwardprovincialunmetmorethanfederalgrant = false&#10;then&#10;  provincialAwardExcessCSGVariation - provincialAwardDisplacedProvincialUnmetNeed&#10;else&#10;  0" target="provincialAwardExcessCSG11" />
+          <zeebe:output source="=max(provincialAwardNetBCSLPotentialAmount - provincialAwardExcessCSG11,0)" target="provincialAwardRemainingBCSL" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0djgmcr</bpmn:incoming>
+      <bpmn:outgoing>Flow_0x3es0p</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:intermediateThrowEvent id="Event_0e39jby" name="awardExcessCSG">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=provincialAwardNetCSLFPotentialAmount + provincialAwardFederalUnmetNeed - provincialAwardNetCSGAmount" target="provincialAwardExcessCSG9" />
```

Comment:
Just double-check if  `provincialAwardFederalUnmetNeed` is  `provincialAwardFederal unmet need`.